selftests/functional/basic.py

```diff
@@ -1101,20 +1118,15 @@ def test_non_absolute_path(self):
 
     @skipUnlessPathExists("/bin/bash")
     def test_env_var_disable(self):
-        with script.TemporaryScript("exec_env_var.sh", EXEC_ENV_VARIABLE_TEST) as tst:
-            res = process.run(
-                f"env TEST_ENV=test avocado-runner-exec-test runnable-run -k exec-test -u {tst}",
-            )
-            result = res.stdout_text.split("\n")[-2]
-            self.assertIn("'returncode': 1", result, "The test might be corrupted.")
+        self._test_env("TEST_ENV=json:null", 0)
 
-            res = process.run(
-                f"env TEST_ENV=test avocado-runner-exec-test runnable-run -k exec-test -u {tst} TEST_ENV=json:null",
-            )
-            result = res.stdout_text.split("\n")[-2]
-            self.assertIn(
-                "'returncode': 0", result, "The env variable was not disabled."
-            )
+    @skipUnlessPathExists("/bin/sh")
+    def test_env_clear_system(self):
+        self._test_env('-c \'{"runner.exectest.clear_env": "system"}\'', 0)
+
+    @skipUnlessPathExists("/bin/sh")
+    def test_env_clear_all(self):
+        self._test_env('-c \'{"runner.exectest.clear_env": "all"}\'', 2)
```

Comment:
I'm not really seeing how this tests `clear_env = all`.  Don't you need a test that fails in the presence of one of the Avocado default environment variables?

Reply:
I agree that this test doesn't bring the full coverage. But because of how this feature is implemented, I think that it is enough to check that one of the Avocado default environment variables has been deleted and assume that all of them has been deleted.

Reply:
So, I agree that testing one of the default variables is removed is good enough, but I'm not seeing where we're doing that - AFAICT the script only checks for `$TEST_ENV` which is not one of the default ones.  Am I missing something?